src/wrapper/mod.rs

```diff
@@ -1,2 +1,23 @@
+use valkey_module::{raw, Context, ContextFlags, ValkeyModuleCtx};
+
 pub mod bloom_callback;
 pub mod defrag;
+
+/// Wrapper for the ValkeyModule_MustObeyClient function.
+/// Takes in an Context and returns true if the if commands are arriving
+/// from the primary client or AOF client and should never be rejected.
+/// False otherwise.
+pub fn is_must_obey_client(ctx: &Context) -> bool {
+    let ctx_raw = ctx.get_raw() as *mut ValkeyModuleCtx;
+    match unsafe { raw::ValkeyModule_MustObeyClient } {
+        Some(func) => {
+            let status = unsafe { func(ctx_raw) as isize };
+            match status {
+                1 => true,
+                0 => false,
+                _ => ctx.get_flags().contains(ContextFlags::REPLICATED),
```

Comment:
This should never be any value other than 0 or 1. 

Instead of silently failing, if this is not 1 or 0, we should know immediately and fix the bug.

So, let us add a panic here
```suggestion
                _ => panic!("We do not expect ValkeyModule_MustObeyClient to return anything other than 1 or 0."),
```